tests/server/server.spec.ts

```diff
@@ -78,14 +95,35 @@ describe('Server', () => {
   test('logs a message when a TCP connection is established', () => {
     hasMockClientConnected = true
     server.openTCPConnection()
-    expect(logMock).toHaveBeenCalledTimes(2)
-    expect(logMock).toHaveBeenCalledWith('Client has connected.')
+    expect(logMock).toHaveBeenCalledTimes(3)
+    expect(logMock).toHaveBeenCalledWith('Client has connected.\n')
```

Comment:
Hmm, this is starting to feel a bit out of hand. It may be time to refactor a bit since really, we're just expecting 1 log to the server not 3.

Reply:
The tests now only check for 1 console log instead of 3. :)